test/plugin_functional/test_suites/core/deprecations.ts

```diff
@@ -87,8 +87,13 @@ export default function ({ getService, getPageObjects }: PluginFunctionalProvide
   ];
 
   describe('deprecations service', () => {
-    before(() => esArchiver.load('test/functional/fixtures/es_archiver/deprecations_service'));
-    after(() => esArchiver.unload('test/functional/fixtures/es_archiver/deprecations_service'));
```

Comment:
What does that change exactly? In both scenarios, the hooks return a promise that is then awaited by the test runner. Manually awaiting the promise doesn't change a thing, unless I'm mistaken?